lib/views/watch_live/watch_live.dart

```diff
@@ -10,15 +12,6 @@ class WatchLive extends StatefulWidget {
 }
 
 class _WatchLiveState extends State<WatchLive> {
-
-  //function to handle 'Join' press
-  void navigateToLobby(){
-    Navigator.push(
-      context,
-      MaterialPageRoute(builder: (context) => const LobbyScreen())
-    );
-  }
```

Comment:
Using the global key & extension type in navigator.dart, we can clean this up a little bit:

```dart
// before
void navigateToLobby(){
  Navigator.push(
    context,
    MaterialPageRoute(builder: (context) => const LobbyScreen())
  );
}

// after
void navigateToLobby() => navigator.push(const LobbyScreen());
```